rest/main_new.go

```diff
@@ -0,0 +1,53 @@
+package rest
+
+import (
+	"flag"
+	"fmt"
+
+	"github.com/couchbase/sync_gateway/base"
+	"github.com/imdario/mergo"
+	pkgerrors "github.com/pkg/errors"
+)
+
+// serverMainPersistentConfig runs the Sync Gateway server with persistent config.
+func serverMainPersistentConfig(osArgs []string, fs *flag.FlagSet, flagStartupConfig *StartupConfig) error {
```

Comment:
It looks like osArgs would be pushed into fs by the time this is called?  Is there a pending use case for osArgs here?

Reply:
Removing until needed.